lib/mittsu/objects/mesh.rb

```diff
@@ -16,6 +16,7 @@ def initialize(geometry = Geometry.new, material = MeshBasicMaterial.new(color:
     end
 
     def update_morph_targets
+      return unless @geometry.respond_to?(:morph_targets)
       if !@geometry.morph_targets.nil? && !@geometry.morph_targets.empty?
```

Comment:
Gosh now I that I look at this whole method, I can't make any sense of it. I'm sure `morph_target_index_by_name` isn't meant to be defined _inside_ of `update_morph_targets` 🤦🏻‍♀️ 
Also the warning string says "Mest" instead of "Mesh" 🫠

Reply:
I was thinking the same 😂